spec.yml

```diff
@@ -356,6 +379,37 @@ components:
           type: string
           example: "jetpack-backup-daily"
 
+    ProductFamilyWithPrices:
+      type: object
+      properties:
+        name:
+          type: string
+          example: "Jetpack Backup"
+        slug:
+          type: string
+          example: "jetpack-backup"
+        products:
+          type: array
+          items:
+            $ref: '#/components/schemas/ProductFamilyWithPricesProduct'
+            
+    ProductFamilyWithPricesProduct:
+      type: object
+      properties:
+        name:
+          type: string
+          example: "Jetpack Backup (Daily)"
```

Comment:
It's veeeeery nitpicky, but should we perhaps use a more up-to-date product as an example since this is going to be replaced with the tiered version at some point in the future?
It could be: 

```
"name": "Jetpack Complete",
"product_id": 1234,
"slug": "jetpack-complete",
"currency": "usd",
"amount": 9.99
```

_(which would also require us to change the product family example as well)_

```
"name": "Jetpack Packs",
"slug": "jetpack-packs",
```